practice-02/src/index.html

```diff
@@ -17,18 +17,115 @@
+                                            class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/samsungnew-220x48-1.png" width="30"
+                                            class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/OPPO42-b_5.jpg" width="30" class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/logo-xiaomi-220x48-5.png" width="30"
+                                            class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/vivo-logo-220-220x48-3.png" width="30"
+                                            class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Realme42-b_37.png" width="30" class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Nokia42-b_21.jpg" width="30" class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Masstel42-b_0.png" width="30" class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Itel42-b_54.jpg" width="30" class="logo" />
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Mobell42-b_19.jpg" width="30" class="logo" />
+                                    </a>
+                                </div>
+                            </div>
+                            <div class="show-total-item">
+                                <div class="show-total-title">Price</div>
+                                <div class="filter-list ">
+                                    <a href="#" class="logo-item"> 2 - 4 million </a>
+                                    <a href="#" class="logo-item"> 4 - 8 million </a>
+                                    <a href="#" class="logo-item"> 8 - 15 million </a>
+                                    <a href="#" class="logo-item"> over 15 million </a>
```

Comment:
always uppercase first character. Please check and update all the same cases

Reply:
Yes, I Changed the first letter of the character ( over ... )